### ISSUE TITLE
Comment editor has keyboard trap

### ISSUE THREAD

**Issue by torgeirl** [devilry/devilry-django#1201]
Already a lot of issues regarding the comment editor (see #1040 for an overview), but it also contains a keyboard trap. That has to be solved in order to pass success Criteria 2.1.2 in WCAG 2.

**Description**
The comment editor uses tab for `\t` and there doesn't seem to be a way to escape the comment field when using only the keyboard for navigation. This is a clear violation with WCAG 2.1.2.

**Comment by espenak** [comment]
Fixed in cradmin-legacy, but we will probably need to make updates for #1182 too, so not released yet.
